docs/src/markdown/contributing.md

```diff
@@ -85,7 +85,7 @@ There are a couple of dependencies that must be present before running the tests
 
 ## Documentation Improvements
 
-A ton of time has been spent not only creating and supporting this plugin, but also spent making this documentation.  If you feel it is still lacking, show your appreciation for the plugin by helping to improve the documentation.  Help with documentation is always appreciated and can be done via pull requests.  There shouldn't be any need to run validation tests if only updating documentation.
+A huge amount of time has been spent not only creating and supporting this plugin, but also redacting this documentation.  If you feel it's still lacking, show your appreciation for the plugin by helping to improve the documentation.  Help with documentation is always appreciated and can be done via pull requests.  There shouldn't be any need to run validation tests when updating just the documentation.
```

Comment:
I'd prefer in some sense in keep the vocabulary a bit more accessible to the common person, so something other than `redacting` would be better.

Reply:
What about "writing the documentation"? The verbs "do" and "make" have a really broad in meaning. 

> PS: Don't get me wrong on such corrections, they are not about showing off vocabulary or being too meticulus, it has to do with the fact that I've studied a bit the psychological aspects of comunication (especially reading) and as a general rule when the reader finds broad terms or undefined statements, in the back of his mind he keeps wondering with questions (which distract him from reading on).
> 
> As an example, if I tell you that I paid "something a lot of money", you'll be left wondering how much "a lot" really is. Similar unanswered question tend to build up and distract the flow of reading, and they also apply to smaller details. Of course, in narrative this might be something desired (where you want to reader to freely fantasize in the interpretation of the text and loose his sense of time), but in technical documentation you're looking for a smooth and linear path with as little bumps on the road as possible, because the reader's goal is about learning and memorizing, and such texts tend to be more cumbersome than other readings.